setup.py

```diff
@@ -13,11 +14,11 @@
       author='Chirag Modi',
       author_email='modichirag@berkeley.edu',
       license='MIT',
-      packages=['flowpm'],
-      install_requires=['astropy', 'scipy', 'mesh-tensorflow'],
-      tests_require=['fastpm'],
+      packages=find_packages(),
+      install_requires=['astropy', 'scipy', 'mesh-tensorflow', 'tensorflow_probability'],
+      tests_require=['fastpm', 'pyccl'],
       extras_require={
-        'testing':  ["fastpm"],
+        'testing':  ['fastpm', 'pyccl'],
```

Comment:
we should try to remove this dependency on pyccl, nbodykit has functions for cosmology computations, we can use those

Reply:
removed dependency on pyccl and switched to nbodykit